todo_app/data/trello_items.py

```diff
@@ -0,0 +1,71 @@
+import requests
+import os
+import dotenv
+
+
+dotenv.load_dotenv()
+
+def get_items():
+    key =  os.getenv("TRELLO_API_KEY")
+    token =  os.getenv("TRELLO_API_TOKEN")
+    conn ="trello-proxy.azure-api.net"
+
+
+    headersList = {
+    "Accept": "*/*",
+    "User-Agent": "Thunder Client (https://www.thunderclient.com)" 
+    }
+
+    response =requests.get ("https://trello-proxy.azure-api.net/1/boards/6575b1e0697c0c91113c2035/lists?key={}&token={}&cards=open&card_fields=id,name".format(key,token),headers= headersList)
+
+    result =response.json()
+    cards=[]
+    for trello_list in response_list:
+        for trello_card in trello_list['cards']:
+            item = Item.from_trello_card(trello_card, trello_list)
```

Comment:
I think we're missing a corresponding import: `from todo_app.item import Item` at the start